nats-base-client/nats.ts

```diff
@@ -505,7 +505,7 @@ export class NatsConnectionImpl implements NatsConnection {
   }
 
   jetstream(
-    opts: JetStreamOptions = {},
+    opts: JetStreamOptions | JetStreamManagerImpl = {},
```

Comment:
this doesn't look right - JetstreamManagerImpl?

Reply:
you are right